docs/docs/ucr.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Universal Container Runtime (UCR) Run Configurations
+---
+
+# UCR
+
+It is now possible to run jobs with Docker containers using [Universal Container Runtime](http://mesos.apache.org/documentation/latest/container-image/).
+
+To enable UCR, the run configuration needs to contain the `ucr` section:
+
+```
+{
+  "description": "example ucr specification",
+  "id": "ucr-docker",
+  "run": {
+    "cmd": "sleep inf",
+    "cpus": 0.2,
+    "mem": 32,
+    "ucr": {
+      "image":  {
+        "id": "ubuntu",
+        "forcePull": true
+      }
+    }
+  }
+}
+```
+
+Please note that UCR uses a differnt syntax when defining docker image section. Also, specifying both `docker` and `ucr` will lead to a validation error.
```

Comment:
I would move this before the example and express it like so:

```markdown
**Important:** Note that UCR could support various image types, therefore the `ucr.image` property is 
an _object_ instead of a _string_ as with `docker`. Specifying just `"image": { "id": "ubuntu" }` will 
default to `docker` image type.
```

And then, _after_ the example, add:

```markdown
Note that specifying both `ucr` and `docker` is invalid and will lead to a validation error.
```

Reply:
I like it! 👍